src/Actions.js

```diff
@@ -0,0 +1,24 @@
+import ActionTypes from './ActionTypes';
+import Dispatcher from './Dispatcher';
+
+export function bootstrapStores(...stores) {
+  let payload = stores.map(store => ({name: store.name, state: store.dehydrate()}));
+  return {
+    type: ActionTypes.BOOTSTRAP_STORES,
+    payload
+  };
+}
```

Comment:
We don't want it go through the dispatcher because this is something we don't need to broadcast to all clients but only for those who are just initialized. At the same time I want it to be look like an action to store because they can receive it on client through dispatcher and handle accordingly...